mail-client/bower/bower-0.9.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="tinfo"
+
+DEPEND="app-crypt/gpgme:=
+	>=dev-lang/mercury-11.07"
+RDEPEND="app-crypt/gpgme:=
+	net-mail/notmuch
+	sys-apps/coreutils
+	sys-libs/ncurses"
```

Comment:
…which reminds me, you want `:0=` here. And you don't want to claim it's not needed at build time when it obviously is.